tests/simple_ubo/simple.cpp

```diff
@@ -129,7 +129,8 @@ int main(int argc, char* argv[])
     bool success = true;
     auto buffer_data = static_cast<cl_int*>(ptr);
     for (cl_uint i = 0; i < BUFFER_SIZE/sizeof(cl_int); ++i) {
-        if (buffer_data[i] != static_cast<cl_int>(i + 1)) {
+        int i_div_4 = i / 4;
+        if (buffer_data[i] != static_cast<cl_int>(i_div_4 + 1)) {
             printf("Failed comparison at buffer_data[%d]: expected %d but got %d\n",
                    i, i + 1, buffer_data[i]);
```

Comment:
Didn't spot you forgot to update the message, I'll fix that as part of folding this test into the framework.